test/e2e/daemon_restart.go

```diff
@@ -247,7 +245,6 @@ var _ = Describe("DaemonRestart", func() {
 	})
 
 	AfterEach(func() {
-		defer framework.afterEach()
 		close(stopCh)
 		expectNoError(DeleteRC(framework.Client, ns, rcName))
```

Comment:
looks like this was the thing we missed... this `AfterEach` needs to be defined before `NewFramework` is called as well.